insights/client/__init__.py

```diff
@@ -95,6 +95,25 @@ def branch_info(self):
         """
         return client.get_branch_info(self.config, self.connection)
 
+    @_net
+    def get_egg_url(self):
+        """
+        Get the proper url based on the configured egg release branch
+        """
+        protocol = "https://"
+        if self.config.insecure_connection:
+            protocol = "http://"
+            self.cert_verify = False
+
+        url = protocol + constants.base_url + constants.module_router_path
```

Comment:
Also using constants.base_url here will force it to use cloud.redhat which will conflict with what's in the config/won't work in Sat/conflicts with the legacy cert